src/pages/partners/components/SponsorIndex.js

```diff
@@ -0,0 +1,55 @@
+import { graphql, useStaticQuery } from "gatsby"
+import { GatsbyImage } from "gatsby-plugin-image"
```

Comment:
We'll need to remove these Gatsby dependencies.

Reply:
Resolved